packages/react-reconciler/src/ReactFiberReconcilerTypes.js

```diff
@@ -0,0 +1,369 @@
+/* eslint-disable */
+type Source = {fileName: string, lineNumber: number};
+type ReactElement = {
+  $$typeof: any,
+  type: any,
+  key: any,
+  ref: any,
+  props: any,
+  _owner: any, // ReactInstance or ReactFiber
+  // __DEV__
+  _store: {validated: boolean},
+  _self: React$Element<any>,
+  _shadowChildren: any,
+  _source: Source,
+};
+type ReactNode =
+  | React$Element<any>
+  | ReactCall<any>
+  | ReactReturn<any>
+  | ReactPortal
+  | ReactText
+  | ReactFragment;
+type ReactFragment = ReactEmpty | Iterable<React$Node>;
+type ReactNodeList = ReactEmpty | React$Node;
+type ReactText = string | number;
+type ReactEmpty = null | void | boolean;
+type ReactCall<V> = {
```

Comment:
I believe most of these types are already included in flow?

Reply:
Maybe. I don't know for sure. This file is auto generated. It's the result of the babel script. Right now it's present in the PR to only give a sense of what the output will look like.